.eslintrc.js

```diff
@@ -171,6 +171,15 @@ module.exports = {
         selector: 'ImportDeclaration[source.value="react"] ImportDefaultSpecifier',
       },
     ],
+
+    // We re-export default in many places
+    'no-restricted-exports': 'off',
```

Comment:
There's an issue in the ESLint repo that tracks this: https://github.com/eslint/eslint/issues/15617. Could be worth adding a comment that we wait for its resolution.

Reply:
👍 Mentioning https://github.com/airbnb/javascript/issues/2500 instead as its closing will enable us to remove the override